javalin/pom.xml

```diff
@@ -91,6 +91,12 @@
             <artifactId>micrometer-core</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>io.jaegertracing</groupId>
+            <artifactId>jaeger-client</artifactId>
+            <version>1.2.0</version>
+        </dependency>
```

Comment:
1. Added this dependency to get access to the jaeger client

2. Is this the right pom file to add it in?

Reply:
You have to add it to the dependency management section in the parent POM, then include it here without version (and with `optional`). The plugin then needs to check if the dependency is included.